### DISCUSSION TITLE
Custom Line Height

### DISCUSSION THREAD

**Discussion by zennethos** [iina/iina#5184] [Ideas]
We already have letter spacing, so it would be great to add line height and word spacing customization, especially line height, as I believe it makes a significant difference.

**Comment by low-batt** [comment]
Where can you specify letter spacing?

**Reply by zennethos** [reply]
settings > subtitle > font spacing

**Comment by low-batt** [comment]
For audio/video playback and subtitle rendering support IINA is using a library from the [mpv](https://github.com/mpv-player/mpv) project. Subtitle rendering is performed by `libmpv`. The IINA `Font spacing` setting is tied to the mpv [--sub-spacing](https://mpv.io/manual/stable/#options-sub-spacing) option.

IINA can only add support for subtitle line height and word spacing if mpv has that capability. Looking through the [Subtitles](https://mpv.io/manual/stable/#subtitles) section of the mpv manual I did not see any controls that provide these features. I could of missed something.

These features need to be implemented by the [mpv project](https://github.com/mpv-player/mpv) before IINA can add support for them.
